backend/routes_announcements.py

```
@@ -0,0 +1,319 @@
+
+            # Start with base query
+            query = Announcement.query
+
+            # Apply filters if provided
+            if priority:
+                query = query.filter(Announcement.priority == priority)
+
+            if active_only:
+                # Only show active announcements that haven't expired
+                now = datetime.utcnow()
+                query = query.filter(Announcement.is_active == True)
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if Announcement.is_active:` for truth checks

Replace with `Announcement.is_active`

(E712)